image_ocr.go

```diff
@@ -25,22 +19,14 @@ func ConvertImage(r io.Reader) (string, map[string]string, error) {
 	defer f.Done()
 
 	meta := make(map[string]string)
-	out := make(chan string, 1)
-
-	// TODO: Why is this done in a separate goroutine when ConvertImage blocks until it returns?
-	go func(file *LocalFile) {
-		langs.RLock()
-		body := gosseract.Must(gosseract.Params{Src: file.Name(), Languages: langs.lang})
-		langs.RUnlock()
-		out <- string(body)
-	}(f)
 
-	return <-out, meta, nil
-}
+	client := gosseract.NewClient()
+	defer client.Close()
+	client.SetImage(f.Name())
+	text, err := client.Text()
+	if err != nil {
+		return "", nil, err
+	}
 
-// SetImageLanguages sets the languages parameter passed to gosseract.
-func SetImageLanguages(l string) {
```

Comment:
This changes the external package interface. Some users may be expecting to use this?

Reply:
agree, reinstated